Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12.7-alpine3.10 AS build
+FROM golang:1.13.0-alpine3.10 AS build
```

Comment:
The latest version is `1.13.6-alpine3.11`.

Reply:
Or maybe we just use `alpine` which always points to the latest version.

Reply:
Thanks. Updated to `1.13.6-alpine3.11`. I think more explicit is better.